tests/integration-tests/tests/dashboard/test_dashboard/test_dashboard/pcluster.config.yaml

```diff
@@ -15,13 +15,17 @@ Scheduling:
       Networking:
         SubnetIds:
           - {{ public_subnet_id }}
+      CustomActions:
+        OnNodeConfigured:
+          # install script that causes issues
+          Script: s3://{{ bucket }}/scripts/preinstall.sh
       ComputeResources:
         - Name: compute-resource1
           {% if scheduler == "awsbatch" %}
           InstanceTypes:
             - {{ instance }}
           {% else %}
-          InstanceType: {{ instance }}
+          InstanceType: t2.micro
```

Comment:
Please change the t2.micro to {{ instance }}

Reply:
changed as per previous comment